src/dashboard/apps/renewable/helpers.py

```diff
@@ -95,3 +96,49 @@ def send_notification_for_opening(entity: Entity) -> None:
         # fail silently and send a sentry log
         sentry_sdk.capture_exception(e)
         return
+
+
+def get_first_station_name(dp: DeliveryPoint) -> str:
+    """Get the name of the first linked station (case-insensitive)."""
+    linked_stations = dp.get_linked_stations()
+    if linked_stations:
+        return list(linked_stations.keys())[0].casefold()
+    return ""
```

Comment:
This may be more explicit?

```suggestion
def get_first_station_name(dp: DeliveryPoint) -> str | None:
    """Get the name of the first linked station (case-insensitive)."""
    linked_stations = dp.get_linked_stations()
    if linked_stations:
        return list(linked_stations.keys())[0].casefold()
    return None
```